test/unit/tech/flash.test.js

```diff
@@ -141,3 +145,35 @@ test('seekable', function() {
   result = seekable.call(mockFlash);
   equal(result.length, mockFlash.duration_, 'seekable is empty with a zero duration');
 });
+
+// fake out the <object> interaction but leave all the other logic intact
```

Comment:
What's your thoughts on this type of mock vs the `mockFlash` object above? Is see that `setCurrentTime` requires a super call, but interested otherwise.